tools/run_tests/run_tests.py

```diff
@@ -139,6 +139,49 @@ def _is_use_docker_child():
   return True if os.getenv('RUN_TESTS_COMMAND') else False
 
 
+def _python_config_generator(name, major, minor, bits, config_args):
+  return PythonConfig(
+    name,
+    config_args.shell + config_args.builder + config_args.builder_prefix_arguments + [
+      _python_pattern_function(major=major, minor=minor, bits=bits)] + [
+      name] + config_args.venv_relative_python + config_args.toolchain,
+    config_args.shell + config_args.runner + [
+      os.path.join(name, config_args.venv_relative_python[0])])
+
+
+def _pypy_config_generator(name, major, config_args):
+  return PythonConfig(
+    name,
+    config_args.shell + config_args.builder + config_args.builder_prefix_arguments + [
+      _pypy_pattern_function(major=major)] + [
+      name] + config_args.venv_relative_python + config_args.toolchain,
+    config_args.shell + config_args.runner + [
+      os.path.join(name, config_args.venv_relative_python[0])])
+
+
+def _python_pattern_function(major, minor, bits):
+  # Bit-ness is handled by the test machine's environment
+  if os.name == "nt":
+    if bits == "64":
+      return '/c/Python{major}{minor}/python.exe'.format(major=major,
```

Comment:
I've been reluctant to point it out because the right answer is "one day [pull request 6544](https://github.com/grpc/grpc/pull/6544) will solve this forever", but since that day hasn't yet come: _favor_ line-wrapping immediately after an opening parenthesis. You'll get to put a lot more on the following line than this other style that has you "stacking" things on the right side of the module text.